.github/workflows/cargo_update.yaml

```diff
@@ -0,0 +1,32 @@
+name: "cargo_update"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # At 7:00am on Wednesday
+    - cron: "0 7 * * 3"
+
+jobs:
+  flake_update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install nix
+        uses: cachix/install-nix-action@v31
+        with:
+          install_url: https://releases.nixos.org/nix/nix-2.28.3/install
+
+      - name: Flake update
+        run: |
+          nix develop .#ci --command cargo update
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v7
```

Comment:
Should this use a token from the secrets that should be available on most repos now for release automation? Then the workflows will run because the change it submitted by a trusted user. I didn't want to maintain tokens by hand across many repos which is why this workflow is set up in other places to use the default token to create the PR.